.gitignore

```diff
@@ -1,3 +1,4 @@
 .Build
 composer.lock
 /var
+.idea
```

Comment:
Having IDE stuff in an repository is hell. Nice that you saw it.

I suggest setting up a global gitignore on your computer. This means that this folder does not have to be explicitly excluded in every project.

https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#:~:text=name%3E-,core.excludesfile,-You
or more readable
https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

```suggestion
```